core/src/OpenMPTarget/Kokkos_OpenMPTarget_Exec.hpp

```diff
@@ -733,13 +734,22 @@ class OpenMPTargetExecTeamMember {
   using space = execution_space::scratch_memory_space;
 
  public:
+  // FIXME: OPENMPTARGET - 8 bytes at the begining of the scratch space for each
```

Comment:
Weren't we using FIXME_OPENMPTARGET as token?
Also typo begining -> beginning

Reply:
I dint realize that. I will replace the token and fix the typo.